rfc/0006-kubewarden-versioning.md

```diff
@@ -0,0 +1,210 @@
+|              |                                                 |
+|:-------------|:------------------------------------------------|
+| Feature Name | Kubewarden versioning                           |
+| Start Date   | Mar 3rd 2022                                    |
+| Category     | Versioning                                      |
+| RFC PR       | [PR4](https://github.com/kubewarden/rfc/pull/4) |
+
+# Summary
+[summary]: #summary
+
+Kubewarden is composed of many core components that work together in
+order to build the final product. This is a proposal to harmonize
+Kubewarden versioning schema.
+
+# Motivation
+[motivation]: #motivation
+
+Having a way to specify what version of Kubewarden is being installed
+or executed would make it easier for the Kubewarden team to describe,
+market and support Kubewarden (as in user support).
+
+It would help users in understanding what version of Kubewarden they
+are about to install or execute.
+
+## Examples / User Stories
+[examples]: #examples
+
+### User story #1
+
+> Users do not need to keep track of component versions and how they
+> work together. They just need to care about running a specific
+> Kubewarden version.
+
+### User story #2
+
+> Users are able to easily understand that a new version of Kubewarden
+> is available, and that more than one component might be upgraded in
+> order to move to the new Kubewarden version.
+
+### User story #3
+
+> A user has an issue with Kubewarden. Given its multiple component
+> nature, we would like to know what was the Kubewarden version that
+> got installed, and through what versions it was upgraded.
+
+### User story #4
+
+> As Kubewarden maintainers we want to define a version skew policy
+> that is easy to be understood and implement.
+> Something similar to what [Kubernetes does](https://kubernetes.io/releases/version-skew-policy/)
+
+# Detailed design
+[design]: #detailed-design
+
+## Many components, one experience
+
+Kubewarden core components, which are exposed to the user are:
+
+- Helm Charts
+- Kubewarden Controller
+- Policy Server
```

Comment:
I'm not sure if we mean the policy-server, or the CRDs. I suppose the former, as policies are not named. If so, I would name it policy-server.